doc/model/precision.md

```diff
@@ -0,0 +1,15 @@
+# Floating-point precision of the model
+
+The following options control the precision of the model:
+
+- The environment variable {envvar}`DP_INTERFACE_PREC` controls the interface precision of the model, the descriptor, and the fitting, the precision of the environmental matrix, and the precision of the normalized parameters for the environmental matrix and the fitting output.
+- The training parameter {ref}`model[standard]/fitting_net[ener]/precision <precision>` controls the precision of neural networks in the descriptor and the fitting, and the subsequent operations after the output of neural networks.
```

Comment:
This ref seems not working. BTW, precisions in descriptor & type embedding(tf) should also be listed here?

Reply:
It seems it should be ```{ref}`precision <model[standard]/fitting_net[ener]/precision>````.